CSS/components/button.css

```diff
@@ -0,0 +1,57 @@
+.btn{
+    min-width: 5rem;
+    min-height: 1.5rem;
+    padding: 0.5rem;
+    background-color: var(--cta);
+    color: var(--button-text-color);
+    border-radius: 0.2rem;
+}
+.btn i{
+    font-size: 1.5rem;
+}
+.btn-secondary{
+    border: 3px solid var(--cta);
+    background-color: var(--background-color);
+    color: var(--headline-color);
+}
+.btn-secondary:hover{
+    background-color: var(--cta);
+    color: var(--button-text-color);
+}
+.btn-success{
+    background-color: var(--success-color);
+    color: var(--success-color-light);
+}
+.btn-failed{
+    background-color: var(--fail-color);
+    color: var(--fail-color-light);
+}
+.btn-suc-outline{
+    border: 3px solid var(--success-color);
+    background-color: var(--success-color-light);
+    color: var(--success-color);
+}
+.btn-suc-outline:hover{
+    background-color: var(--success-color);
+    color: var(--success-color-light);
+}
+.btn-fail-outline{
+    border: 3px solid var(--fail-color);
+    background-color: var(--fail-color-light);
+    color: var(--fail-color);
+}
+.btn-fail-outline:hover{
+    background-color: var(--fail-color);
+    color: var(--fail-color-light);
+}
+.btn-icon{
+    padding: 1rem;
+}
+.fab-div{
+    position: relative;
+}
+.fab{
+    min-width: 4rem;
```

Comment:
fab should have a position fixed relative to the body.

Reply:
I was also trying to use the fixed the btn but I think if we leave users to decide the position accordingly to their way of organising the HTML will be fine bz I was having this problem while testing 
I forget to remove the .fab-div will remove it thank you for pointing out